src/pages/en/core-concepts/routing.md

```diff
@@ -94,6 +94,14 @@ In this example, a request for `/withastro/astro/tree/main/docs/public/favicon.s
 }
 ```
 
+## Excluding pages
+
+You can exclude pages, or even whole directories from being built by prefixing their names with an underscore (`_`).
+
+This allows for co-locating tests, utilities, and components with the pages they're used in without them being built into html files and outputted into the `dist/` directory like everything else.
+
+For example, a file named `src/pages/_hidden-page.astro` would not show up in the build output, and both `src/pages/_hidden-directory/page-1.astro` and `src/pages/_hidden-directory/page-2.astro` would not show up in the build output.
```

Comment:
I would prefer if this was a tree overview of the `src/pages/` directory, for it would summarise the concepts that are encapsulated into the page and would fit nicely into this section.

```tree
src/pages/
 ├──  _tests/       #Private and excluded from build
  |       └──test1.astro
 ├──  blog/        
  |       ├── [...posts].astro
  |       └── index.astro
 └──  index.astro
 ```

Reply:
Agree with the tree, and now feel vindicated since @aFuzzyBear had the same thought! Go Team!

@zadeviggers - Can you work with my tree above (I think it shows a bunch of different types of things to be excluded, which is the point) and adjust it to be more realistic, if you need to?

Reply:
Okay, will do!